Sources/DistributedActors/WeakActorDictionary.swift

```diff
@@ -14,8 +14,70 @@
 
 import Distributed
 
-struct WeakActorDictionary {
-    var underlying: [ClusterSystem.ActorID: WeakDistributedActorContainer] = [:]
+/// A dictionary which only weakly retains the
+public struct WeakActorDictionary<Act: DistributedActor>: ExpressibleByDictionaryLiteral
```

Comment:
TODO: incomplete; need to make them a bit nicer API so we can use them in examples ourselfes as well